src/zenml/integrations/kubeflow/orchestrators/kubeflow_orchestrator.py

```diff
@@ -77,17 +80,120 @@ class KubeflowOrchestrator(BaseOrchestrator):
 
     custom_docker_base_image_name: Optional[str] = None
     kubeflow_pipelines_ui_port: int = DEFAULT_KFP_UI_PORT
-    kubernetes_context: Optional[str] = None
+    kubernetes_context: str = ""
```

Comment:
Can't we even leave this without a default value?

Reply:
Not if I'm using a root validator with `pre=False`, which is something I want, because then I get the other attributes already validated, including the UUID. This is somewhat of a pattern that is being used in at least 2 other places (airflow and mlflow).

Reply:
Argh, I just realized I wanted to do the same (use a root validator) for those other places, but I lost those changes somewhere along the way.

Reply:
In fact, I have to go back and make this optional again, because I'm breaking backwards-compatibility if I don't.

Reply:
Yes, I even tried using `pre=True`, but the UUID is generated on creation and not yet available during that phase.